setup.cfg

```diff
@@ -30,7 +30,8 @@ install_requires =
     azure-mgmt-resource >= 15
     boto3 >= 1.14.20
     botocore >= 1.17.20
-    google-api-python-client >= 1.7.7
+    google-cloud-compute <= 1.9.0
+    googleapis-common-protos == 1.63.1
```

Comment:
given that you had issues with dependency resolution, could you add a comment to each of these entries explaining why they are pinned the way they are?

Reply:
Good call. I actually should test without pinning `google-cloud-compute` now that I found the issue lied with the `googleapis-common-protos` dependency.